src/order/filter/cofinite.lean

```diff
@@ -90,3 +90,28 @@ end
 lemma nat.frequently_at_top_iff_infinite {p : ℕ → Prop} :
   (∃ᶠ n in at_top, p n) ↔ set.infinite {n | p n} :=
 by simp only [← nat.cofinite_eq_at_top, frequently_cofinite_iff_infinite]
+
+lemma filter.tendsto.exists_forall_le {α β : Type*} [nonempty α] [linear_order β] [no_top_order β]
```

Comment:
Do you really need the `not_top_order` assumption? I guess the result should hold in general, right?

Reply:
Thanks! Removed the assumption.